lib/git.js

```diff
@@ -427,14 +428,30 @@ class Git extends EventEmitter {
    * @method listen
    * @memberof Git
    * @param  {Number}   port     - the port to start the server on
+   * @param  {Array}   port     - the https options
    * @param  {Function} callback - the function to call when server is started or error has occured
    */
-  listen(port, callback) {
-      var self = this;
-      this.server = http.createServer(function(req, res) {
+  listen(port, options, callback) {
+    if(typeof options == 'function') {
+      callback = options;
+    }
+
+    var self = this;
```

Comment:
can this be hoisted towards the top of the function like before?

Reply:
Fixed